docs/release/updating-to-osg-24.md

```diff
@@ -6,6 +6,61 @@ Updating to OSG 24
 [OSG 24](release_series.md#series-overviews) (the *new series*) introduces support for the ARM architecture. Changes
 required to upgrade from OSG 23 are relatively minor.
 
+-   [Compute Entrypoints](../compute-element/htcondor-ce-overview.md) should be updated to OSG 24 as soon as possible.
+
+-   [HTCondor pools](#updating-your-htcondor-hosts) and [access points](#updating-your-osg-access-point) should be
+    updated to OSG 24 as soon as possible.
+
+-   All other services (e.g., OSG Worker Node clients, Frontier Squids) should be updated to
+    [OSG 24](#updating-the-osg-repositories) as soon as possible.
+
+Updating the OSG Repositories
+-----------------------------
+
+!!! note
+    Before updating the OSG repository, be sure to turn off any OSG services. Consult the sections below
+    that match your situation.
```

Comment:
```suggestion
!!! note
    Because configuration updates will be necessary, be sure to turn off any OSG services
    before updating them. Consult the sections below that match your situation.
```
and also move it before the 'yum update' command.  There is no need to turn off services just to update the repos.

Reply:
Moved